corehq/apps/data_interfaces/views.py

```diff
@@ -667,19 +667,14 @@ def post(self, request, *args, **kwargs):
     def edit_url_name(self):
         return EditCaseRuleView.urlname
 
-    @property
-    @memoized
```

Comment:
kind of a shame we don't cache `get_timezone_for_user`, but feel free to call that out of scope

Reply:
Yeah I'm going to pull the out of scope card, though I did create [a ticket](https://dimagi-dev.atlassian.net/browse/SAAS-14137) for it.

Reply:
that counts as something